functions/delete-all/1.1/index.js

```diff
@@ -0,0 +1,76 @@
+import templayed from '../../utils/templayed';
+
+const getAll = async (modelName, where, queryBody) => {
+  const queryName = `all${modelName}`;
+  const getAllQuery = `
+    query {
+      ${queryName}(where: ${where}, take: 200) {
+        ${queryBody}
+      }
+    }
+  `;
+
+  const { data, errors } = await gql(getAllQuery);
+
+  if (errors) {
+    throw errors;
+  }
+
+  const {
+    [queryName]: { results, totalCount },
+  } = data || {};
+
+  return { results, totalCount };
+};
+
+const deleteBatch = async (modelName, where) => {
+  const { results = [] } = await getAll(modelName, where, 'results { id }');
+  const ids = results.map((item) => item.id);
+  const deleteMutation = `
+    mutation($input: ${modelName}Input) {
+      deleteMany${modelName}(input: $input) {
+        id
+      }
+    }
+  `;
+
+  if (ids.length) {
+    const { errors } = await gql(deleteMutation, { input: { ids } });
```

Comment:
deleting is quite fast, but the actual requests isnt. Maybe you can combine multiple read requests with this delete to speed up this process?

Reply:
Of course you could do that, but this feature request is not about optimizing the speed. I would suggest you to create another feature request or ticket on your backlog.

Reply:
I dont understand it , you can just pass the variableMap to this function right ?!